### PR TITLE
Add the capability to change state strings

### PR DESCRIPTION
**Pull request by aacebedo**
This PR enables to change different strings used in sway effects.

### CODE REVIEW

**Comment by aacebedo** [comment]
up ?

**Comment by aacebedo** [comment]
ping ?

**Comment by aacebedo** [comment]
@jirutka ? can you merge this please ?

File: completions/fish/swaylock.fish [hunk 1]
```diff
@@ -53,3 +54,5 @@ complete -c swaylock -l text-ver-color              --description "Sets the colo
 complete -c swaylock -l text-wrong-color            --description "Sets the color of the text when invalid."
 complete -c swaylock -l tiling                 -s t --description "Same as --scaling=tile."
 complete -c swaylock -l version                -s v --description "Show the version number and quit."
+complete -c swaylock -l verstr                      --description The string displayed when the password is being verified."
+complete -c swaylock -l wrongstr                    --description The string displayed when the password is wrong."
```
Comment (jirutka) [inline completions/fish/swaylock.fish:58]: There’s a missing opening quote mark in each description you’ve added.

File: completions/zsh/_swaylock [hunk 2]
```diff
@@ -4,56 +4,59 @@
...
+  '(--show-keyboard-layout -k)'{--show-keyboard-layout,-k}'[Display the current xkb layout while typing]' \
+  '(--text-caps-lock-color)'--text-caps-lock-color'[Sets the color of the text when Caps Lock is active]:color:' \
+  '(--text-clear-color)'--text-clear-color'[Sets the color of the text when cleared]:color:' \
+  '(--text-color)'--text-color'[Sets the color of the text]:color:' \
+  '(--text-ver-color)'--text-ver-color'[Sets the color of the text when verifying]:color:' \
+  '(--text-wrong-color)'--text-wrong-color'[Sets the color of the text when invalid]:color:' \
+  '(--tiling -t)'{--tiling,-t}'[Same as --scaling=tile]' \
+  '(--version -v)'{--version,-v}'[Show the version number and quit]' \
+  '(--verstr)'--verstr'[The string displayed when the password is being verified.]:string:' \
+  '(--wrongstr)'--wrongstr'[The string displayed when the password is wrong.]:string:'
```
Comment (jirutka) [inline completions/zsh/_swaylock:62]: Please never mix formatting and functional changes in a single commit. And don’t introduce any formatting changes here.

**Pull request closed without merging**